src/vcpkg/vcpkgpaths.cpp

```diff
@@ -720,8 +808,9 @@ namespace vcpkg
         if (!fs.exists(destination))
         {
             const fs::path destination_tmp = destination_parent / fs::u8path("baseline.json.tmp");
-            auto treeish = Strings::concat(commit_sha, ":versions/baseline.json");
-            auto maybe_contents = git_show(treeish, this->root / fs::u8path(".git"));
+            auto relative_path = fs.relative(VCPKG_LINE_INFO, versions_dir, dot_git_dir.parent_path());
```

Comment:
the versions directory is _always_ at `/versions`.

Reply:
No, maybe you have:
root
- .git 
- my_registry
   - ports
   - versions

Or do we not support this, so the registry must be at the root of a git repo?

Reply:
We do not support this.